extensions/apicurio-registry-avro/deployment/src/main/java/io/quarkus/apicurio/registry/avro/ApicurioRegistryDevServicesBuildTimeConfig.java

```diff
@@ -29,7 +29,7 @@
      * The Apicurio Registry image to use.
      * Note that only Apicurio Registry 2.x images are supported.
      */
-    @ConfigItem(defaultValue = "docker.io/apicurio/apicurio-registry-mem:2.1.5.Final")
+    @ConfigItem(defaultValue = "docker.io/apicurio/apicurio-registry-mem:2.2.0.Final")
```

Comment:
Hum, I suppose this _has_ to be defined in this "static" way? Asking because of #23428.

Reply:
We can definitely improve this, although it is outside the scope of this PR.